src/Umbraco.Core/Configuration/Models/RichTextEditorSettings.cs

```diff
@@ -124,7 +120,8 @@ public class RichTextEditorSettings
     /// <summary>
     ///     HTML RichText Editor TinyMCE Plugins
     /// </summary>
-    public string[] Plugins { get; set; } = Default_plugins;
+    [DefaultValue(StaticPlugins)]
+    public IEnumerable<string> Plugins { get; set; } = StaticPlugins.Split(',');
```

Comment:
As this is a public method on a public class, this would be a breaking change, so this would have to wait until at least version 11.

Reply:
Yes, it would be a breaking change. Unfortunately I couldn't see this makes a difference. So not sure why the schema is generated different for `DisallowedUploadFiles` and `AllowedUploadFiles` properties.